src/session/content/item_row.rs

```diff
@@ -94,18 +96,358 @@ impl ItemRow {
                 match item.type_() {
                     ItemType::Message(message) => {
                         let content = message.content().0;
-                        let message = message.to_owned().upcast();
+                        let sender_name = match message.sender().to_owned() {
+                            MessageSender::User(data) => super::item_row::sender_name(&data),
+                            MessageSender::Chat(data) => data.title(),
+                        };
+                        let message_obj = message.to_owned().upcast();
 
                         match content {
                             MessageContent::MessagePhoto(_) => {
-                                self.set_child_row::<MessagePhoto>(message)
+                                self.set_child_row::<MessagePhoto>(message_obj)
                             }
                             MessageContent::MessageSticker(data) if !data.sticker.is_animated => {
-                                self.set_child_row::<MessageSticker>(message)
+                                self.set_child_row::<MessageSticker>(message_obj)
+                            }
+                            MessageContent::MessageChatChangeTitle(data) => {
+                                let title = data.title;
+                                let child = self.get_or_create_event_row();
+                                let type_ = message.chat().type_().clone();
+                                match type_ {
+                                    ChatType::Supergroup(data) => {
+                                        let is_channel = data.is_channel();
+                                        if is_channel {
+                                            child.set_label(
+                                                gettext!(
+                                                    "<b>Channel name was changed to «{}»</b>",
+                                                    title
+                                                )
+                                                .trim(),
```

Comment:
Why you're using `trim()` here (and also in the rest of the code below)? It seems that there's nothing to trim in these strings anyway.